hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/schema/SendSchemaMessageTask.java

```diff
@@ -74,4 +84,11 @@ public String getMethodName() {
     public Object[] getParameters() {
         return null;
     }
+
+    private Collection<UUID> getCurrentMemberUuids() {
+        return nodeEngine.getClusterService().getMembers()
+                .stream()
+                .map(Member::getUuid)
+                .collect(Collectors.toList());
```

Comment:
I know that the member list returned here and the member list that participated in the replication process can differ. Consider this the temporary solution until the PR for the API-1441 is sent, where we will handle this.